src/libpmemfile-posix/file.c

```diff
@@ -359,6 +359,20 @@ _pmemfile_openat(PMEMfilepool *pfp, struct pmemfile_vinode *dir,
 		goto end;
 	}
 
+	if (flags & PMEMFILE_O_PATH)
```

Comment:
In the next commit ("posix: clear SUID & SGID when writing to a file owned by someone else", which GitHub shows as earlier one for some reason) I need access to file->flags at earlier moment. I'll move this chunk of code to the relevant commit and fix ordering.